src/modules/fancyzones/lib/Zone.cpp

```diff
@@ -80,11 +80,24 @@ void Zone::SizeWindowToZone(HWND window, HWND zoneWindow) noexcept
         OffsetRect(&zoneRect, mi.rcMonitor.left - mi.rcWork.left, mi.rcMonitor.top - mi.rcWork.top);
     }
 
-    WINDOWPLACEMENT placement;
+    WINDOWPLACEMENT placement{};
     ::GetWindowPlacement(window, &placement);
     placement.rcNormalPosition = zoneRect;
     placement.flags |= WPF_ASYNCWINDOWPLACEMENT;
-    placement.showCmd = SW_RESTORE | SW_SHOWNA;
+    placement.showCmd &= (~SW_SHOWNORMAL) & (~SW_SHOWMINIMIZED);
```

Comment:
What is this for? Can you add a comment to explain, otherwise in the future is someone else has to maintain this code they will have to figure it out.

Reply:
Added some comments, PTAL